playground/mocks/config/responseConfig.js

```diff
@@ -417,6 +417,20 @@ const windowAuthnLoopbackFailfast = {
   ],
 };
 
```

Comment:
maybe should name it as MacOS instead of windows, since we are technically supporting this feature for MacOS